modules/seqexec/server/src/main/scala/seqexec/server/ghost/GHOSTController.scala

```diff
@@ -39,11 +100,40 @@ final case class GHOSTController[F[_]: Sync](ghostClient: GHOSTClient[F],
 }
 
 object GHOSTController {
+  sealed abstract class BundleConfig(val configName: String)
+  object BundleConfig {
+    case object Standard extends BundleConfig("lo")
+    case object HighRes extends BundleConfig("hi")
+  }
+
+  sealed abstract class IFUNum(val ifuNum: Int) {
+    val ifuStr: String = s"ghost:cc:cu:ifu$ifuNum"
+  }
+  object IFUNum {
+    case object IFU1 extends IFUNum(1)
+    case object IFU2 extends IFUNum(ifuNum = 2)
+  }
+
+  sealed abstract class IFUTargetType(val targetType: Int)
+  object IFUTargetType {
+    case object NoTarget extends IFUTargetType(0)
```

Comment:
probably using named params here too?

Reply:
Sure. I'm not sure why one ended up named and not the others. I'll make them all named.